src/test/java/api/loans/InHouseUseCheckInTest.java

```diff
@@ -120,4 +128,33 @@ public void isNotInHouseUseWhenCheckInServicePointIsNotServingHomeLocation() {
     assertThat(checkInResponse.getJson().containsKey("item"), is(true));
     assertThat(checkInResponse.getInHouseUse(), is(false));
   }
+
+  private void verifyLastCheckInWasRecorded(UUID itemId, UUID servicePoint) {
+    final CqlQuery query = queryFromTemplate("itemId=%s and itemStatusPriorToCheckIn=Available",
+      itemId);
+    final MultipleJsonRecords recordedOperations = checkInOperationClient.getMany(query);
+
+    assertTrue(recordedOperations.totalRecords() > 0);
```

Comment:
Why was this approach chosen over using a `greaterThan` matcher?

Reply:
I've changed to use the method. Just was not aware of it.